sdk/tables/data-tables/samples-dev/createAndDeleteEntities.ts

```diff
@@ -9,28 +9,26 @@
  */
 
 import { TableClient, TablesSharedKeyCredential } from "@azure/data-tables";
-import { v4 } from "uuid";
 // Load the .env file if it exists
 import * as dotenv from "dotenv";
 dotenv.config();
 
 const tablesUrl = process.env["TABLES_URL"] || "";
 const accountName = process.env["ACCOUNT_NAME"] || "";
 const accountKey = process.env["ACCOUNT_KEY"] || "";
-const tableSufix = v4().replace(/-/g, "");
 
 async function createAndDeleteEntities() {
   console.log("== Create and delete entities Sample ==");
 
   // Note that this sample assumes that a table with tableName exists
-  const tableName = `createAndDeleteEntitiesTable${tableSufix}`;
+  const tableName = `createAndDeleteEntitiesTable`;
```

Comment:
I think the above comment is no longer valid, since the sample will create the table if it does not exist?